File: auro/src/app/@core/data/zahtjevi-redovni-otpis-detalji.ts
export interface ZahtjeviRedovniOtpisDetalji {
  odobreniArtikli: Array< {
  artikalID: number,
  brojOtpisa: string;
  sifra: string;
  razlogOtpisa: string;
  kolicina: number;
  nazivArtikla: string;
  status: string;
  datumPopunjavanja: string;
  dobavljac: string;
  ukupnaVrijednost: number;
  nabavnaVrijednost: number;}>,

  odbijeniArtikli: [
    {
      odobreniArtikli: Array< { sifra: string;
        naziv: string;
        razlogOtpisa: string;
        potrebnoZbrinjavanje: string;
        potrebanTransport: string;
        jedinicaMjere: string;
        kolicina: number;
        nabavnaVrijednost: number;
        ukupnaVrijednost: number;
        dobavljac: string;}>;

        odbio: string;
        komentar: string;
    }
  ]

}

File: backend/Models/ZahtjeviRedovniOtpisDetalji.cs
using Microsoft.EntityFrameworkCore;

namespace backend.Models {
    public class ZahtjeviRedovniOtpisDetalji {
       // public string BrojOtpisa { get; set; } = null!;
       public int ArtikalId {get; set;}
        public string Sifra { get; set; } = null!;
        public string RazlogOtpisa { get; set; } = null!;

        [Precision(18,2)]
        public decimal Kolicina { get; set; }
        public string NazivArtikla { get; set; } = null!;
        public string DatumPopunjavanja { get; set; } = null!;
        public string Status { get; set; } = null!;
        public string Dobavljac { get; set; } = null!;

        [Precision(18, 2)]
        public decimal NabavnaVrijednost { get; set; }

        [Precision(18, 2)]
        public decimal UkupnaVrijednost { get; set; }
        
    }
}